Cllog/Projects/CllogService/CoreKit/DesignKit/Sources/Component/Button/GeneralButton.swift

```diff
@@ -116,9 +116,8 @@ public extension GeneralButton {
 
 // MARK: - Preview
 #Preview {
-    ClLogFont.register()
     
-    return VStack {
```

Comment:
Cannot use explicit 'return' statement in the body of result builder 'ViewBuilder'